src/Gui/Components/AutomatedScrollingFrame.luau

```diff
@@ -0,0 +1,35 @@
+local Hoarcekat = script:FindFirstAncestor("Hoarcekat")
+local React = require(Hoarcekat.Packages.React)
+
+local e = React.createElement
+
+function AutomatedScrollingFrame(props)
+	local canvasSize, updateCanvasSize = React.useBinding(Vector2.new())
+
+	local resize = React.useCallback(function(rbx)
```

Comment:
I prefer `instance`

Reply:
Component to be deleted, resolving this change.